assets/scss/components/_navbar.scss

```diff
@@ -1,46 +1,261 @@
+    transform: rotateX(0deg);
+  }
+}
+
+.spacer {
+  width: 100%;
+  height: 195px;
+}
+
+#item1,
+#item2,
+#item3 {
+  display: inline;
+}
+
+#actual_content1 {
+  color: black;
+  position: fixed;
+  -webkit-transition: all 600ms ease-out;
+  -moz-transition: all 600ms ease-out;
+  -o-transition: all 600ms ease-out;
+  -ms-transition: all 600ms ease-out;
+  transition: all 600ms ease-out;
+  margin-top: 275px;
+  margin-left: -200px;
+  opacity: 0;
+  z-index: 1;
+
+  &:hover {
+    z-index: 3 !important;
+    opacity: 1;
+  }
+}
+
+#actual_content2 {
+  color: black;
+  position: fixed;
+  -webkit-transition: all 600ms ease-out;
+  -moz-transition: all 600ms ease-out;
+  -o-transition: all 600ms ease-out;
+  -ms-transition: all 600ms ease-out;
+  transition: all 600ms ease-out;
+  margin-top: 275px;
+  margin-left: -200px;
+  opacity: 0;
+  z-index: 1;
+
+  &:hover {
+    z-index: 3 !important;
+    opacity: 1;
+  }
+}
+
+#actual_content3 {
+  color: black;
+  position: fixed;
+  -webkit-transition: all 600ms ease-out;
+  -moz-transition: all 600ms ease-out;
+  -o-transition: all 600ms ease-out;
+  -ms-transition: all 600ms ease-out;
+  transition: all 600ms ease-out;
+  margin-top: 275px;
+  margin-left: -200px;
+  opacity: 0;
+  z-index: 1;
+
+  &:hover {
+    z-index: 3 !important;
+    opacity: 1;
+  }
+}
+
+#dormant_content {
+  position: fixed;
+  width: 101%;
+  height: 200px;
+  margin-top: 275px;
+  margin-left: -450px;
+  z-index: 2;
+}
+
+#item1:hover ~ {
+  #actual_content1 {
+    z-index: 3 !important;
+    -webkit-transition: all 400ms ease-in;
+    -moz-transition: all 400ms ease-in;
+    -o-transition: all 400ms ease-in;
+    -ms-transition: all 400ms ease-in;
+    transition: all 400ms ease-in;
+    opacity: 1;
+    display: block;
+  }
+}
+
+#item2:hover ~ {
+  #actual_content2 {
+    z-index: 3 !important;
+    -webkit-transition: all 400ms ease-in;
+    -moz-transition: all 400ms ease-in;
+    -o-transition: all 400ms ease-in;
+    -ms-transition: all 400ms ease-in;
+    transition: all 400ms ease-in;
+    opacity: 1;
+    display: block;
+  }
+}
+
+#item3:hover ~ {
+  #actual_content3 {
+    z-index: 3 !important;
+    -webkit-transition: all 400ms ease-in;
+    -moz-transition: all 400ms ease-in;
+    -o-transition: all 400ms ease-in;
+    -ms-transition: all 400ms ease-in;
+    transition: all 400ms ease-in;
+    opacity: 1;
+    display: block;
+  }
+}
+
+.item_block {
+  -webkit-transition: all 500ms ease-out;
+  -moz-transition: all 500ms ease-out;
+  -o-transition: all 500ms ease-out;
+  -ms-transition: all 500ms ease-out;
+  transition: all 500ms ease-out;
+  display: grid;
+  width: 300px !important;
+  height: 150px !important;
+  margin-right: 50px;
+  border-color: black;
+  border-width: 2px;
+  border-style: solid;
+
+  &:hover {
+    transform: scale(1.01);
+    box-shadow: 0 0 3px #515151;
+  }
+}
+
+.left-item {
+  width: 150px;
+  height: 150px;
+  border-right: 2px solid black;
+}
+
+//Possible arrow?
```

Comment:
Fun fact, in scss you can do /* */ syntax to comment stuff out! Dont feel the need to change it, I just wanted to let you know because its a cool feature of scss.

Reply:
Cool, thanks!